tuf/ngclient/updater.py

```diff
@@ -262,10 +292,12 @@ def _download_metadata(
     ) -> bytes:
         """Download a metadata file and return it as bytes"""
         encoded_name = parse.quote(rolename, "")
-        if version is None:
-            url = f"{self._metadata_base_url}{encoded_name}.json"
+
+        self.spec_version += ".0.0"
```

Comment:
You shouldn't need to do this. We may want to update the test data instead.

Reply:
Could you reiterate on what you mean by "updating the test data"? Do you mean updating the data in `tests/repository_data` ?
Also one more reason to not use this would be because major versions are being referenced with the digit only, eg: 1.0.0 is being referenced as 1.

Reply:
> Do you mean updating the data in tests/repository_data/ ?

Yes, exactly.